### ISSUE TITLE
Project Checkpoint 3 is Ready for Review

### ISSUE THREAD

**Issue by ClassyElm** [ClassyElm/MinecraftJSONGenerator#17]
Hey, @pawaitemadisoncollege! I believe everything is ready to go for checkpoint 3.

In case you need it, you can access the project here:
http://3.19.205.136:8080/jsongenerator/

I have noticed that my EC2 instance on AWS has been sending alerts about CPU usage, so if you cannot access the page, just let me know. I might have to restart the instance.

I should add that, while the web application does display data from the database, you cannot save commands to accounts through the website yet. For that reason, I created a dummy account for you to use that has some commands manually added. Here are the credentials if you would like to test it:

Username: jsmitty@example.com
Password: password
(The best password)

The account only has the user access role, so I am not concerned about sharing this information publicly. Also, you can try deleting that user's commands if you want. 😄

If you want to create a new account, feel free to give it a go.

**Comment by pawaitemadisoncollege** [comment]
Hey @ClassyElm Can you give that instance a bounce?  Do you have the status check auto-bounce option set up and it's just not doing the trick? 

- [ ] It looks like you've been making good use of branches and pull requests. Do you feel like that flow is working well? How do you decide how much to merge at a time? Is it by feature, or by "ah, this is a bunch of working stuff that I should merge".  No judgement, I'm just curious about your process.

**Comment by ClassyElm** [comment]
I restarted the instance. That should work now. I'll have to look into the auto-bounce option again.

As for the work flow of merging, I like it a lot. At first, I saw it as just another step for making changes in the repository, but I do like it as a means to control versions between other developers and main vs development builds. Typically, I merge after each session of coding (when I remember to). Sometimes I wait until I feel the branch is in a functioning state. That's my mindset at least.
